KubeArmor/core/kubeArmor.go

```diff
@@ -616,7 +616,7 @@ func KubeArmor() {
 		dm.Logger.Print("Started to monitor container security policies on gRPC")
 	}
 
-	if !cfg.GlobalCfg.K8sEnv && cfg.GlobalCfg.HostPolicy {
+	if !dm.K8sEnabled {
```

Comment:
Wouldn't this enable Host Policy even if HostPolicy is disabled?

Reply:
No, we will receive the host policy but it won't be enforced.

Reply:
But we shouldn't be receiving the host policy :)

Reply:
Actually we are checking again if hostPolicy is enabled when creating the apparmor profile and applying it. Here: https://github.com/kubearmor/KubeArmor/blob/main/KubeArmor/enforcer/appArmorEnforcer.go#L588